### PR TITLE
Pendant: Template: Single

### LINKED ISSUE #5744: Pendant: Templates: Single Post
image

image

Edit `single.html` to achieve the design.  Ensure it works both with/out featured images.

### PR DESCRIPTION
**Pull request by pbking**
May fix: #5744 
Started by @PeterNdomano [here](https://github.com/Automattic/themes/pull/5757).

This has a dependence on #5755 which should come in first.

#### Changes proposed in this Pull Request:

Adds an increased Header size to achieve target design for Single Page.
(Note: The [Page Title] block renders an H2 element, but the design called for the size shown for H1.  To compensate I increased the size of H1 and backfilled the sizes from H4.  This way the natively rendered H2 element hits the design.  Please let me know if that's OK for the design.

I have also converted all of the units from px to rems in this process.

Before:
image
image

After:
image
image

Everything else was achievable using either Global Styles or Block-level configuration in the FSE.

Gap sizes are approximate compared to the figma design, however the implementation is simply constructed and scales dynamically.  Hopefully the spacing suits.

Desktop With Featured Image
image
image

Mobile With Featured Image
image

Desktop Without Featured Image
image

Mobile Without Featured Image

image

### CODE REVIEW

File: pendant/theme.json [hunk 1]
```diff
@@ -1,295 +1,310 @@
 {
-    "version": 2,
-    "templateParts": [
-        {
-            "name": "header",
-            "area": "header"
-        },
```
Comment (MaggieCabrera) [inline pendant/theme.json:7]: Is this happening because of the spaces to tabs change?

Comment (pbking) [reply pendant/theme.json:7]: Yes.

**Comment by pbking** [comment]
Going to bring this in but leave the associated issue open for final design review.

File: pendant/theme.json [hunk 2]
```diff
@@ -1,297 +1,310 @@
...
+		}
+	},
+	"styles": {
+		"blocks": {
+			"core/button": {
+				"border": {
+					"radius": "0px"
+				},
+				"color": {
+					"background": "var:preset|color|foreground",
```
Comment (colorful-tones) [inline pendant/theme.json:201]: @pbking @MaggieCabrera would either of you be able to please point me to how/what this syntax is doing please? `var:preset|color|foreground`

**Pull request merged**